pom.xml

```diff
@@ -401,8 +403,90 @@
                 </plugins>
             </build>
         </profile>
+        <!--
```

Comment:
1. We need the profile to exclude the JS dependency as you no longer need it. document that
2. We need a profile for isolates with all platforms in case you want to build a platform indepent build.
3. We should be able to run the tests with isolated profiles.

Reply:
Fixed by https://github.com/graalvm/polyglot-embedding-demo/pull/12/commits/16f309fd5d89a51c5db57ee48aa8b9ec4fe0189f.